Measures/CustomExportParams/measure.xml

```diff
@@ -0,0 +1,191 @@
+<measure>
```

Comment:
Do we need this measure? it does not seem to do anything?

Reply:
This is the dummy measure for certain parameters, which are not required for performing a simulation, but are necessary for the import/export functionality. That said, some of these additional parameters are placed in other measure simply due to the history of the code. Perhaps in the future we can move all of these to this measure. (Added to tasks)